### ISSUE TITLE
Mobile Entry: cursor arrows don't work

### ISSUE THREAD

**Issue by pjanx** [fyne-io/fyne#5258]
### Checklist

- [X] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same problem, before opening a new one.
- [X] This issue only relates to a single bug. I will open new issues for any other problems.

### Describe the bug

Cursor arrows do not work on Entries on either Android or iOS simulator.
This may be a duplicate/subissue of #2331.

### How to reproduce

* Select an Entry.
* Write something in it.
* Try to use either of:
   * real keyboard arrows:
      * connect a USB keyboard to an Android or iOS device over USB
      * type arrows into an Android or iOS emulator
   * virtual keyboard arrows, such as these on Android:
      * Unexpected Keyboard \*
      * Microsoft SwiftKey Japanese \**
      * Samsung Keyboard Japanese

### Screenshots

\* That one button provides cursor arrows
![Screenshot_20241116_120240](https://github.com/user-attachments/assets/75ca160e-f98c-4578-b3d4-7781cb4ce835)

\** The `<` `>` buttons are cursor arrows
![Screenshot_20241116_120111](https://github.com/user-attachments/assets/f29d97a5-b3a5-4976-bea1-555f091108d4)

### Example code

```
package main

import (
	"fyne.io/fyne/v2"
	"fyne.io/fyne/v2/app"
	"fyne.io/fyne/v2/widget"
)

func main() {
	a := app.New()
	w := a.NewWindow("Bugs")
	w.Resize(fyne.NewSize(640, 480))
	w.SetContent(widget.NewMultiLineEntry())
	w.ShowAndRun()
}
```

### Fyne version

2.5.2

### Go compiler version

1.23.2

### Operating system and version

Android

**Comment by andydotxyz** [comment]
How do you get virtual arrow keys up? Please include in replication steps.

**Comment by pjanx** [comment]
Updated, with screenshots.

**Comment by andydotxyz** [comment]
Thanks.
This seems to be android specific so I updated title as well.

**Comment by pjanx** [comment]
Actually, no, you can connect any USB keyboard to an iPhone 15. And then there's iOS Simulator as well.

**Comment by andydotxyz** [comment]
> Even just normal iPad keyboards have arrows, although I don't have that either.

Can you tell me how to get that up on an iPad? I cannot see them...

**Comment by andydotxyz** [comment]
Confirmed with bluetooth keyboard connected to iOS as well

**Comment by pjanx** [comment]
By normal iPad keyboards, I meant, e.g. the Magic Keyboard.

**Comment by andydotxyz** [comment]
Thanks, fixed now on develop and will be in 2.5.3
